qa/tasks/vstart_runner.py

```diff
@@ -987,12 +1000,46 @@ def __init__(self):
     def __del__(self):
         shutil.rmtree(self.teuthology_config['test_path'])
 
+def _teardown_cluster():
+    log.info('\ntearing down the cluster...')
+    remote.run(args = [os.path.join(SRC_PREFIX, "stop.sh")], timeout=60)
+    remote.run(args = ['rm', '-rf', './dev', './out'])
+
+def _clear_old_logs():
+    from os import stat
+    from sys import version_info
+
+    global logpath
+    # TODO: get rid of python2 part when it deprecates.
+    if version_info.major >= 3:
+        exception = FileNotFoundError
```

Comment:
Realistically, no one is ever going to run this code. Anyone using `vstart_runner.py` must also be running it with `python2`. You might as well always assume python2 in testing and not bother with compatibility unless its trivial and backwards/forwards compatible.

Reply:
I thought we would make it python3 compatible sooner or later.

Reply:
That might happen but it will be a complete and atomic transition. We won't have a situation where test code may be run with either py2 or py3.

Leaving this compatibility switch in is not worthwhile in my opinion for the simple reason that no one will ever exercise that code path until the py3 conversion occurs. When that happens, this code may not work correctly anyway. Best to leave it out.